truss/cli/cli.py

```diff
@@ -351,7 +353,6 @@ def _create_chains_table(service) -> Tuple[rich.table.Table, List[str]]:
     required=False,
     help="Name of the remote in .trussrc to push to.",
 )
-@error_handling
```

Comment:
I assume you removed this, because the stack trace is useful (required?) to interpret some of the framework validation errors.

It was generally a bit of a pending question to me how the UX for displaying such information in the CLI should be like.

* For errors specifically, I would ideally like to move away from stack traces and instead have an error collector class that contextualizes errors and gives output more like a linter (we have a ticket here BT-10488, it's quite a bit of work todo though...)
* Logging: I think the default log formatting is not very nice, prefixing all lines with `INFO:root`. I personally like absl-style logging with timestamp, file path, function and line number (often very useful information). But for streamlining the CLI we might even want something less verbose/technical - WDYT?

Reply:
I actually did not mean to remove this. I'm going to add it back for now, but I agree with you where the way we display errors is really not good.

Let's think about that and handle this better.